cg/cli/workflow/rnafusion/base.py

```diff
@@ -104,6 +105,7 @@ def run(
     revision: str,
     compute_env: str,
     use_nextflow: bool,
+    id: Optional[int],
```

Comment:
```suggestion
    nf_tower_run_id: Optional[int],
```
Or is it just tower_id?

Reply:
yes! I'll change it